pandas/core/tools/datetimes.py

```diff
@@ -759,13 +760,20 @@ def to_datetime(
         <https://docs.python.org/3/library/datetime.html
         #strftime-and-strptime-behavior>`_ for more information on choices, though
         note that :const:`"%f"` will parse all the way up to nanoseconds.
+        You can also pass:
+
+        - "ISO8601", to parse any `ISO8601 <https://en.wikipedia.org/wiki/ISO_8601>`_
+          time string (not necessarily in exactly the same format);
+        - "mixed", to infer the format for each element individually. This is risky,
+          and you should probably use it along with `dayfirst`.
```

Comment:
It's true that in that case that is best, but the typicaly caveat of that this `dayfirst=True/False` is not strict still applies, right?

Reply:
true, but it would the typical example at least
```
In [2]: pd.to_datetime(['12-01-2000 00:00:00', '13-01-2000 00:00:00'], format='mixed', dayfirst=True)
Out[2]: DatetimeIndex(['2000-01-12', '2000-01-13'], dtype='datetime64[ns]', freq=None)
```

Reply:
Yes, I was just wondering if it would be useful to still call it out here explicitly (but it's quite explicit in the `dayfirst` entry in the docstring)